source/basic.tex

```diff
@@ -3099,8 +3099,7 @@
 inside a nested struct declaration and the other is not, or if the two are
 separated by a zero-length bit-field declaration, or if they are separated by a
 non-bit-field declaration. It is not safe to concurrently update two bit-fields
-in the same struct if all fields between them are also bit-fields of nonzero
-width.
+in the same struct if all members between them are also bit-fields of nonzero width.
```

Comment:
A bit-field of zero width is not a (data) member, so I think this is semantics-changing if we assume "fields" includes bit-fields of zero width.